src/coreclr/scripts/superpmi.md

```diff
@@ -116,14 +118,23 @@ The above command collects over all of the managed code called by the
 child process. Note that this allows many different invocations of any
 managed code.
 
-You can also collect using PMI instead of running code. Do with with the `--pmi` and `-pmi_assemblies`
+You can also collect using PMI instead of running code. Do with with the `--pmi` and `-assemblies`
 arguments. E.g.:
 
 ```
-python f:\gh\runtime\src\coreclr\scripts\superpmi.py collect --pmi -pmi_assemblies f:\assembly_store -output_mch_path f:\collections\my_collection.mch
+python f:\gh\runtime\src\coreclr\scripts\superpmi.py collect --pmi -assemblies f:\assembly_store -output_mch_path f:\collections\my_collection.mch
 ```
 
-Note that collection generates gigabytes of data. Most of this data will
+You can alternatively collect by running `crossgen` on a set of assemblies, e.g.:
+
+```
+python f:\gh\runtime\src\coreclr\scripts\superpmi.py collect --crossgen -assemblies f:\assembly_store -output_mch_path f:\collections\my_collection.mch
+```
+
+You can specify both `--pmi` and `--crossgen` to do both types of collections across the
```

Comment:
Any reason we want to do this? I think we should differentiate between pmi and crossgen runs?

Reply:
It seems unlikely you'd want this, but it just falls out and maybe it will be useful